components/discord/actions/send-message-advanced/send-message-advanced.mjs

```diff
@@ -5,7 +5,7 @@ export default {
   key: "discord-send-message-advanced",
   name: "Send Message (Advanced)",
   description: "Send a simple or structured message (using embeds) to a Discord channel",
-  version: "0.0.1",
+  version: "1.0.0",
```

Comment:
```suggestion
  version: "0.1.0",
```

Reply:
I incremented the major version for these actions since the return value's structure has changed, which could be a breaking change.
Previously, this action returned a [discord.js Message](https://discord.js.org/#/docs/discord.js/main/class/Message) object. In this PR, this action returns either a Discord API message or `{ success: true }` object on success.

Should we keep the major version bump so users are alerted that the action may have breaking changes if they choose to update it in a workflow?

Reply:
Makes sense to me, Jacob